src/main/java/com/iexec/commons/containers/client/DockerClientInstance.java

```diff
@@ -801,6 +802,56 @@ public synchronized boolean removeContainer(String containerName) {
             return false;
         }
     }
+
+    /**
+     * Retrieves the execution duration of a container.
+     * If the container has not been started yet or has not ended yet,
+     * then return {@link Optional#empty()}.
+     * Otherwise, return the duration.
+     * <p>
+     * /!\ Docker inspection command precision could lead to sub-zero execution duration for fast containers.
+     * In this case, this method return a zero-duration object.
```

Comment:
```suggestion
     * then returns {@link Optional#empty()}.
     * Otherwise, returns the duration.
     * <p>
     * /!\ Docker inspection command precision could lead to sub-zero execution duration for fast containers.
     * In this case, this method returns a zero-duration object.
```